core/src/main/java/com/facebook/testing/screenshot/internal/RandomAccessFileOutputStream.java

```diff
@@ -0,0 +1,24 @@
+package com.facebook.testing.screenshot.internal;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.RandomAccessFile;
+
+class RandomAccessFileOutputStream extends OutputStream {
```

Comment:
This class is now not used, please remove.

Reply:
removed, thanks